src/queries/siteMeta.ts

```diff
@@ -0,0 +1,22 @@
+import { useStaticQuery, graphql } from 'gatsby'
```

Comment:
just to understand - where is it used?

Reply:
It's used [here](https://github.com/iterative/dvc.org/blob/blog-merging/src/components/Page/DefaultSEO/index.tsx#L6) and [here](https://github.com/iterative/dvc.org/blob/blog-merging/src/components/SEO/index.tsx#L5).